common/app/templates/headerInlineJS/membershipAccess.scala.js

```diff
@@ -14,8 +14,9 @@
 
         require([
             'common/utils/$',
-            'common/utils/ajax'
-        ], function($, ajax) {
+            'common/utils/ajax',
+            'common/modules/identity/api'
```

Comment:
This should be correct, yes. The membershipAccess script should run after all the relevant bundles have been loaded, I believe. So by the time this code runs, there will already be ajax and bonzo modules registered to the names you're calling.